test/regressions/index.test.js

```diff
@@ -1,6 +1,6 @@
-import * as path from 'path';
-import * as fse from 'fs-extra';
-import * as playwright from 'playwright';
+const path = require('path');
+const fse = require('fs-extra');
+const playwright = require('playwright');
```

Comment:
I believe the issue is that you are using `@mui/internal-test-utils/setupBabelPlaywright` inside `test/regressions/.mocharc.js` instead of `@mui/internal-test-utils/setupBabel`

Reply:
The playwright specific config seems unnecessary to me btw, maybe we should remove it @michaldudak

Reply:
Changing this to `setupBabel` causes the tests in Base UI to explode, so it may require a bit more work. I'd love to get rid of webpack and mocha and replace them with vite and vitest at some point.

Reply:
Ah, but this seems to work in core. I'm still having trouble with vitest on X, but the next step would be to try to use the same tools everywhere indeed.